### DISCUSSION TITLE
Ways to get bookmarks into Bookmarks on iOS (Brave browser)?

### DISCUSSION THREAD

**Discussion by don-ferris** [nextcloud/bookmarks#2142] [Q&A] [answered]
I need to preface this question with the disclaimer that I haven’t yet installed this so, if there’s a simple, obvious answer that I’d know if I had it installed, the I apologize. Anyway, I do the majority of my browsing (and everything else, like writing this post) on my iPad Pro 12.9”. Very few browser extensions for Chrome, etc. work on iPads since Apple mandates that all browsers be WebKit-based. The workaround is to use an iOS Shortcut. The easiest way to do that is if an app has already included features/functionality that specifically supports shortcuts. Another way is using URLs. Does Bookmarks give me the capability of (e.g.) “https://bookmarks.myserver.com/send-to-bookmarks?url=[url of webpage I want to bookmark]? Another possibility would be to email into Bookmarks. If not, are there any iOS users out there who can tell me how to make this happen?

**Comment by marcelklehr** [comment, marked as answer]
Sorry for the delay. You could use the Floccus iOS app
